getgather/api/api.py

```diff
@@ -35,6 +36,29 @@ def custom_generate_unique_id(route: APIRoute) -> str:
 api_app.include_router(link_router)
 
 
+class StationPage(BaseModel):
+    name: str
+    path: str
+    label: str
+
+
+class StationConfig(BaseModel):
+    pages: list[StationPage]
+
+
+@api_app.get("/station")
+def station() -> StationConfig:
+    pages = [
+        StationPage(path="", label="Get Started", name="GetStarted"),
+        StationPage(path="activities", label="Activities", name="Activities"),
+        StationPage(path="replay", label="Replay", name="Replay"),
+        StationPage(path="mcp-docs", label="MCP Docs", name="McpDocs"),
+    ]
+    if not settings.multi_user_enabled:
+        pages.append(StationPage(path="live-view", label="Live View", name="LiveView"))
+    return StationConfig(pages=pages)
+
+
```

Comment:
I think since the goal is just to enable/disable a page or pass config to the frontend, we don’t really need the server to tell the frontend what to render. Something like [vite-plugin-runtime-env](https://github.com/micha149/vite-plugin-runtime-env) should be enough. wdyt?

Reply:
@broerjuang Will this work with the production bundle? We don't run Vite in the container, so I'm curious what the mechanics will be to get the env variable.